core/src/serde.rs

```diff
@@ -265,24 +283,24 @@ pub struct OracleContractParametersSerde {
 impl From<OracleContractParameters> for OracleContractParametersSerde {
     fn from(p: OracleContractParameters) -> Self {
         OracleContractParametersSerde {
-            p2s: p.p2s.to_base58(),
+            p2s: AddressEncoder::new(NetworkPrefix::Mainnet).address_to_str(&p.p2s),
```

Comment:
I think that eventually, we should switch to base16 serialized ErgoTree for storing contracts in the config files. I mean, the odds of someone setting the new contracts with a testnet prefix are super tiny but non-zero.
Of course, it's out of the scope of this PR.

Reply:
Made an issue for this: https://github.com/ergoplatform/oracle-core/issues/98